client_test.go

```diff
@@ -79,6 +79,70 @@ func startServer(t *testing.T, onConnect func(net.Conn), onMessage func(string))
 	return host
 }
 
+func startNoTLSServer(t *testing.T, onConnect func(net.Conn), onMessage func(string)) string {
+	host := ":" + strconv.Itoa(startPort)
```

Comment:
I would prefer if the host specifies that it's a local connection (so `"127.0.0.1:" + ...`) instead